rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/tsdb/20_mapping.yml

```diff
@@ -111,6 +111,39 @@ top level dim object:
                               type: double
                               time_series_metric: gauge
 
+---
+match object but no match object children:
+  - skip:
+      version: " - 8.0.99"
+      reason: introduced in 8.1.0
```

Comment:
`index.routing_path` is introduced in 8.1.0, and I don't change the error message. But I change the rule, It's a small 
 break change, is it need to change the introduced version to 8.2.0?

Reply:
yeah - if you changed the message you should bump it.

Reply:
ok, I change the version to 8.2.0